### ISSUE TITLE
ERR_MODULE_NOT_FOUND

### ISSUE THREAD

**Issue by zouws** [JustasMonkev/mcp-accessibility-scanner#12]
Hi 

After I installed the server. Once I tried to start it will got failed like below:
`$ npx mcp-accessibility-scanner
node:internal/modules/esm/resolve:265
    throw new ERR_MODULE_NOT_FOUND(
          ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module 'D:\software\nodejs\node_cache\_npx\03d14fe5aa9b614e\node_modules\mcp-accessibility-scanner\lib\program.js' imported from D:\software\nodejs\node_cache\_npx\03d14fe5aa9b614e\node_modules\mcp-accessibility-scanner\cli.js
    at finalizeResolution (node:internal/modules/esm/resolve:265:11)
    at moduleResolve (node:internal/modules/esm/resolve:933:10)
    at defaultResolve (node:internal/modules/esm/resolve:1169:11)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)
    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:227:38)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:87:39)
    at link (node:internal/modules/esm/module_job:86:36) {
  code: 'ERR_MODULE_NOT_FOUND',
  url: 'file:///D:/software/nodejs/node_cache/_npx/03d14fe5aa9b614e/node_modules/mcp-accessibility-scanner/lib/program.js'
}

Node.js v20.16.0`

And can not found the program.js in the direcorty. I tried uninstall and install again still failed.
